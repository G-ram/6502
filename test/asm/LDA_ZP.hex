a2 04 8e 40 00 a5 40 8d 00 01 
