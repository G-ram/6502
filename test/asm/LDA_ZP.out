A: 04
X: 04
Y: 00
P: 00010000
S: 20
PC: 60a
100 : 04
101 : 00
